ui.tests/test-module/specs/datepicker/datepicker.runtime.persian.locale.cy.js

```diff
@@ -43,9 +43,9 @@ describe("Form Runtime with Date Picker", () => {
         cy.get(`#${datePicker8}`).find(".cmp-adaptiveform-datepicker__calendar-icon").should("be.visible").click().then(() => {
             let todayDate = new Date();
             cy.get(".dp-caption").invoke("text").should("eq", "مارس, 2024");
-            cy.get(".dp-caption").click();
+            cy.get(".dp-caption").eq(3).click();
```

Comment:
Changing existing test, means that this is backward incompatible change. What was the reason for adding this ?

Reply:
Hi, @rismehta! Previously, whenever we used to add date-time picker widget to the form, the calendar element associated to widget is initialised and created only once even if there are multiple instances of the widget, i.e., same calendar element being initialised to every unique instance of date-time picker widget which was conflicting when interacting with each of it. 

Now since there are multiple calendar elements, in the tests I had to specify which element the click() event had to occur on, hence the changes.